server/Arcadia.Assistant.Calendar/SickLeave/SendEmailSickLeaveActor.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Arcadia.Assistant.Calendar.SickLeave.Events;
+using System;
+using Akka.Actor;
+using Akka.DI.Core;
+using MimeKit;
```

Comment:
styling. Most of the people prefer using's on top, some of them (including me) - inside namespace. There are arguments for each of these styles, but one thing remains - it should be consistent within project :)